konfuzio_sdk/trainer/document_categorization.py

```
@@ -1456,6 +1454,8 @@ def build_categorization_ai_pipeline(
     categorization_pipeline.category_vocab = categorization_pipeline.build_template_category_vocab()
     # Configure image and text models
     if image_model is not None:
+        if isinstance(image_model, str):
```

Can we simplify this using "ImageModel.__members__.get(string)" ?
Can we raise an exception if there is no match

---

added the exceptions. 
it seems there's no way of getting the element by the value with the enum; I can remake it into a simpler data structure or leave it as it is now.